mftool.sh

```
@@ -43,10 +44,39 @@ function use_origin() {
+    # update
+    git checkout main 
+    git fetch upstream
+    git rebase upstream/main
+    git submodule update --init --remote ./muse
+
+    # switch to fork
+    use_fork
+
+    # create branch
+    git checkout -b $name
+    cd muse
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)